internals/search.py

```diff
@@ -87,9 +87,10 @@ def process_starred_me_query() -> list[int]:
 
 def process_recent_reviews_query() -> list[int] | Future:
   """Return features that were reviewed recently."""
-  query = Gate.query(
-      Gate.state.IN(Gate.FINAL_STATES))
-  future_feature_ids = query.fetch_async(projection=['feature_id'], limit=40)
+  query = Vote.query(Vote.state.IN(Gate.FINAL_STATES))
+  query = query.filter(
+      Vote.set_on > datetime.datetime.now() - datetime.timedelta(days=90))
```

Comment:
Clarify the precedence of operators

```suggestion
      Vote.set_on > (datetime.datetime.now() - datetime.timedelta(days=90)))
```